### DISCUSSION TITLE
Am i using the date picker component incorrectly or does it use two different date formats?

### DISCUSSION THREAD

**Discussion by carlost** [uswds/uswds#6084] [General]
As far as I can tell you need to set the value of the date picker via `data-default-value` using the ISO 8601 format, but the input sends the date using a zero padding MM/DD/YYYY format common in the us. That forces the application to serialize and deserialize the dates to handle the formatting differences - which makes this component more painful to use than necessary.

If that is correct, why not just use a single format (iso or us date format) for the default value & returned value.

**Comment by mahoneycm** [comment]
Hello @carlost

I believe this approach was taken because ISO 8601 is [universally supported date time string format for JavaScript](https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-date-time-string-format). We use this format for our date values and JS calculations but we expect users to input this value in US date format since that is the standard in the US.
